modules/distr-tracing-document-attributes.adoc

```diff
@@ -14,15 +14,15 @@
 
 :DTProductName: Red Hat OpenShift distributed tracing
 :DTShortName: distributed tracing
-:DTProductVersion: 2.0
+:DTProductVersion: 2.1
 
 :JaegerName: Red Hat OpenShift distributed tracing platform
 :JaegerShortName: distributed tracing platform
-:JaegerVersion: 1.28.0
+:JaegerVersion: 1.30.1
```

Comment:
@JStickler JaegerVersion: 1.30.0
```
$ oc get csv
NAME                             DISPLAY                                                 VERSION   REPLACES   PHASE
jaeger-operator.v1.30.0          Red Hat OpenShift distributed tracing platform          1.30.0               Succeeded
opentelemetry-operator.v0.42.0   Red Hat OpenShift distributed tracing data collection   0.42.0               Succeeded
```

Reply:
@jkandasa thanks, I'll update the version.